terraform-continuous-deployment/terraform-continuous-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+description: Automatically run a Terraform plan when a GitHub PR is merged
+
+triggers:
+- name: github-pr-merge
+  source:
+    type: webhook
+    image: relaysh/github-trigger-pull-request-merged:latest
+  binding:
+    parameters:
+      repository: !Data repository
+      branch: !Data branch
+  when:
```

Comment:
FWIW, this isn't implemented yet. Might want to leave it out to prevent confusion for now.

Reply:
RUT ROW